doc/pbh/pbh-design.md

```diff
@@ -602,7 +868,10 @@ config
      |
      |--- rule
      |    |--- add <table_name> <rule_name> OPTIONS
-     |    |--- update <table_name> <rule_name> OPTIONS
+     |    |--- update
+     |    |    |--- field
+     |    |         |--- set <table_name> <rule_name> OPTIONS
```

Comment:
To confirm that set here means that if the field is not present it is added, and if present, field value is replaced with what is provided?

Reply:
@anish-n you are right

Reply:
Ok understood thanks